packages/expect/__typechecks__/expect.test.ts

```diff
@@ -9,5 +9,5 @@ import type * as expect from 'expect';
 
 export type M = expect.Matchers<void, unknown>;
 export type N = expect.Matchers<void>;
-// @ts-expect-error
-export type E = expect.Matchers<>;
+// @ts-expect-error: Generic type 'Matchers<R, T>' requires between 1 and 2 type arguments.
+export type E = expect.Matchers;
```

Comment:
This was my mistake. `// @ts-expect-error` was silencing syntax error instead of expected type error. Wrapping the code into `expect()` would help to avoid mistakes like this one. Just another motivation for Jestish typecheck runner. Working on it.